<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>390</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>351</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="hasilLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>341</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Aplikasi Image Processing</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Citra Asli</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Citras Hasil</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>814</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_Load_Image"/>
    <addaction name="action_Save_Image"/>
   </widget>
   <widget class="QMenu" name="menuPre_Processing">
    <property name="title">
     <string>Operasi Titik</string>
    </property>
    <widget class="QMenu" name="menuContrast">
     <property name="title">
      <string>Contrast</string>
     </property>
     <addaction name="actionSimple_Contrast"/>
     <addaction name="actionAuto_Contrast_2"/>
    </widget>
    <widget class="QMenu" name="menuHistogram">
     <property name="title">
      <string>Histogram</string>
     </property>
     <addaction name="actionGrayscale_Histogram"/>
     <addaction name="actionRGB_Histogram"/>
    </widget>
    <addaction name="actionGrayscale"/>
    <addaction name="actionBrightness"/>
    <addaction name="menuContrast"/>
    <addaction name="actionNegative_Image"/>
    <addaction name="actionBiner_Image"/>
    <addaction name="menuHistogram"/>
    <addaction name="actionPerataan_Histogram"/>
   </widget>
   <widget class="QMenu" name="menuOpearsi_Geometri">
    <property name="title">
     <string>Operasi Geometri</string>
    </property>
    <widget class="QMenu" name="menuRotasi">
     <property name="title">
      <string>Rotasi</string>
     </property>
     <addaction name="action_min_45_Derajat"/>
     <addaction name="action_Min_90_Derajat"/>
     <addaction name="action_plus_45Derajat"/>
     <addaction name="action180_Derajat"/>
     <addaction name="action_plus_90_Derajat"/>
    </widget>
    <widget class="QMenu" name="menuResize">
     <property name="title">
      <string>Resize</string>
     </property>
     <addaction name="actionLinear_Interpolation"/>
     <addaction name="actionCubic_Interppolation"/>
     <addaction name="actionSkewed_Size"/>
    </widget>
    <addaction name="actionTranslasi"/>
    <addaction name="menuRotasi"/>
    <addaction name="actionTranspose"/>
    <addaction name="actionCroping"/>
    <addaction name="menuResize"/>
   </widget>
   <widget class="QMenu" name="menuOperasi_Aritmatika">
    <property name="title">
     <string>Operasi Aritmatika</string>
    </property>
    <addaction name="actionPenambahan_Citra"/>
    <addaction name="separator"/>
    <addaction name="actionLogika_Not"/>
    <addaction name="actionLogika_AND"/>
   </widget>
   <widget class="QMenu" name="menuOperasi_Spasial">
    <property name="title">
     <string>Analisis Citra</string>
    </property>
    <widget class="QMenu" name="menuEdge_Detection">
     <property name="title">
      <string>Edge Detection</string>
     </property>
     <addaction name="actionSobel"/>
     <addaction name="actionPrewitt"/>
     <addaction name="actionRobert"/>
     <addaction name="actionCanny"/>
    </widget>
    <widget class="QMenu" name="menuMorphology">
     <property name="title">
      <string>Morphology</string>
     </property>
     <addaction name="actionDilation"/>
     <addaction name="actionErosion"/>
     <addaction name="actionOpening"/>
     <addaction name="actionClosing"/>
     <addaction name="actionThinning"/>
     <addaction name="actionThickening"/>
     <addaction name="actionHit_or_Miss_Transform"/>
    </widget>
    <addaction name="menuEdge_Detection"/>
    <addaction name="menuMorphology"/>
   </widget>
   <widget class="QMenu" name="menuSegmentation">
    <property name="title">
     <string>Segmentation</string>
    </property>
    <addaction name="actionThresholding_Adaptive"/>
    <addaction name="actionTemplate_Matching"/>
    <addaction name="actionColor_Segmentation"/>
   </widget>
   <widget class="QMenu" name="menuOperasi_Spasial_2">
    <property name="title">
     <string>Operasi Spasial</string>
    </property>
    <addaction name="actionSmoothing_Image"/>
    <addaction name="actionSharpening_Image"/>
    <addaction name="actionMean"/>
    <addaction name="separator"/>
    <addaction name="actionMedian"/>
    <addaction name="actionMax_Filter"/>
    <addaction name="actionMin_Filter"/>
    <addaction name="separator"/>
    <addaction name="actionTransformasi_Fourier"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPre_Processing"/>
   <addaction name="menuOpearsi_Geometri"/>
   <addaction name="menuOperasi_Aritmatika"/>
   <addaction name="menuOperasi_Spasial_2"/>
   <addaction name="menuOperasi_Spasial"/>
   <addaction name="menuSegmentation"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Load_Image">
   <property name="text">
    <string>Load Image</string>
   </property>
  </action>
  <action name="action_Save_Image">
   <property name="text">
    <string>Save Image</string>
   </property>
  </action>
  <action name="actionGrayscale">
   <property name="text">
    <string>Grayscale</string>
   </property>
  </action>
  <action name="actionBrightness">
   <property name="text">
    <string>Brightness</string>
   </property>
  </action>
  <action name="actionAuto_Contrast">
   <property name="text">
    <string>Auto Contrast</string>
   </property>
  </action>
  <action name="actionSimple_Contrast">
   <property name="text">
    <string>Simple Contrast</string>
   </property>
  </action>
  <action name="actionAuto_Contrast_2">
   <property name="text">
    <string>Contrast Stretching</string>
   </property>
  </action>
  <action name="actionNegative_Image">
   <property name="text">
    <string>Negative</string>
   </property>
  </action>
  <action name="actionBiner_Image">
   <property name="text">
    <string>Biner Image</string>
   </property>
  </action>
  <action name="actionPerataan_Histogram">
   <property name="text">
    <string>Histogram Equalization</string>
   </property>
  </action>
  <action name="actionGrayscale_Histogram">
   <property name="text">
    <string>Grayscale Histogram</string>
   </property>
  </action>
  <action name="actionRGB_Histogram">
   <property name="text">
    <string>RGB Histogram</string>
   </property>
  </action>
  <action name="actionTranslasi">
   <property name="text">
    <string>Translasi</string>
   </property>
  </action>
  <action name="actionTranspose">
   <property name="text">
    <string>Transpose</string>
   </property>
  </action>
  <action name="actionCroping">
   <property name="text">
    <string>Cropping</string>
   </property>
  </action>
  <action name="actionPenambahan_Citra">
   <property name="text">
    <string>+-*/</string>
   </property>
  </action>
  <action name="actionPengurangan_Citra">
   <property name="text">
    <string>Pengurangan Citra</string>
   </property>
  </action>
  <action name="actionPerkalian_Citra">
   <property name="text">
    <string>Perkalian Citra</string>
   </property>
  </action>
  <action name="actionPembagian_Citra">
   <property name="text">
    <string>Pembagian Citra</string>
   </property>
  </action>
  <action name="actionLogika_AND">
   <property name="text">
    <string>AND/OR/XOR</string>
   </property>
  </action>
  <action name="actionLogika_OR">
   <property name="text">
    <string>Logika OR</string>
   </property>
  </action>
  <action name="actionLogika_Not">
   <property name="text">
    <string>Invert</string>
   </property>
  </action>
  <action name="actionBitshift">
   <property name="text">
    <string>Bitshift</string>
   </property>
  </action>
  <action name="actionSobel">
   <property name="text">
    <string>Sobel</string>
   </property>
  </action>
  <action name="actionPrewitt">
   <property name="text">
    <string>Prewitt</string>
   </property>
  </action>
  <action name="actionRobert">
   <property name="text">
    <string>Robert</string>
   </property>
  </action>
  <action name="actionCanny">
   <property name="text">
    <string>Canny</string>
   </property>
  </action>
  <action name="actionDilation">
   <property name="text">
    <string>Dilation</string>
   </property>
  </action>
  <action name="actionErosion">
   <property name="text">
    <string>Erosion</string>
   </property>
  </action>
  <action name="actionOpening">
   <property name="text">
    <string>Opening</string>
   </property>
  </action>
  <action name="actionClosing">
   <property name="text">
    <string>Closing</string>
   </property>
  </action>
  <action name="actionThinning">
   <property name="text">
    <string>Thinning</string>
   </property>
  </action>
  <action name="actionThickening">
   <property name="text">
    <string>Thickening</string>
   </property>
  </action>
  <action name="actionHit_or_Miss_Transform">
   <property name="text">
    <string>Hit or Miss Transform</string>
   </property>
  </action>
  <action name="actionThresholding_Adaptive">
   <property name="text">
    <string>Thresholding Adaptive</string>
   </property>
  </action>
  <action name="actionTemplate_Matching">
   <property name="text">
    <string>Template Matching</string>
   </property>
  </action>
  <action name="actionColor_Segmentation">
   <property name="text">
    <string>Color Segmentation</string>
   </property>
  </action>
  <action name="actionSmoothing_Image">
   <property name="text">
    <string>Filter Gaussian</string>
   </property>
  </action>
  <action name="actionSharpening_Image">
   <property name="text">
    <string>Filter Laplace</string>
   </property>
  </action>
  <action name="actionMedian">
   <property name="text">
    <string>Median</string>
   </property>
  </action>
  <action name="actionMean">
   <property name="text">
    <string>Mean Filter</string>
   </property>
  </action>
  <action name="actionTransformasi_Fourier">
   <property name="text">
    <string>Transformasi Fourier</string>
   </property>
  </action>
  <action name="action_min_45_Derajat">
   <property name="text">
    <string>-45 Derajat</string>
   </property>
  </action>
  <action name="action_Min_90_Derajat">
   <property name="text">
    <string>-90 Derajat</string>
   </property>
  </action>
  <action name="action_plus_45Derajat">
   <property name="text">
    <string>+45 Derajat</string>
   </property>
  </action>
  <action name="action180_Derajat">
   <property name="text">
    <string>180 Derajat</string>
   </property>
  </action>
  <action name="action_plus_90_Derajat">
   <property name="text">
    <string>+90 Derajat</string>
   </property>
  </action>
  <action name="actionLinear_Interpolation">
   <property name="text">
    <string>Linear Interpolation</string>
   </property>
  </action>
  <action name="actionCubic_Interppolation">
   <property name="text">
    <string>Cubic Interppolation</string>
   </property>
  </action>
  <action name="actionSkewed_Size">
   <property name="text">
    <string>Skewed Size</string>
   </property>
  </action>
  <action name="actionMin_Filter">
   <property name="text">
    <string>Min Filter</string>
   </property>
  </action>
  <action name="actionMax_Filter">
   <property name="text">
    <string>Max Filter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
